File: program/find-perimeter-of-a-rectangle/FindPerimeterOfARectangle.ts
function FindPerimeterOfARectangle(length: number, width: number) {
    // Perimeter fo rectangle = 2(l*w);

    let perimeter = 2*(length + width);
    return perimeter;
}

console.log("Perimeter of rectangle is ", FindPerimeterOfARectangle(5, 2)+"cm");

File: program/find-perimeter-of-a-rectangle/FindPerimeterOfARectangle.kt
import java.util.Scanner

fun main() {

    val number1 = Scanner(System.`in`)
    println("Enter Length: ")
    val length: Double = number1.nextDouble()

    val number2 = Scanner(System.`in`)
    println("Enter Width: ")
    val width: Double = number2.nextDouble()

    val perimeter = 2 * (length + width)
    println("The Perimeter of Rectangle is $perimeter")
}
